streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -273,12 +273,14 @@ private long processNext(final List<ConsumerRecord<byte[], byte[]>> records,
         long nextPosition = -1;
         int numberRecords = records.size();
         int numberRestored = 0;
+        long lastRestoredOffset = 0;
```

Comment:
Nit: init with `-1` (does not really change much, but indicates "unset" better than zero).

Reply:
ack